### PR TITLE
feat: Tabbing Order E2E Tests - Product List 

### PR DESCRIPTION
**Pull request by Zeyber**
Closes: #4201

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts [hunk 1]
```diff
@@ -1202,4 +1204,980 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
...
+    {
+      value: '»',
+      type: TabbingOrderTypes.LINK,
+    },
+    {
+      type: TabbingOrderTypes.CX_PRODUCT_VIEW,
+    },
+    {
+      value: '/electronics-spa/en/USD/product/3514520/DSC-WX1',
+      type: TabbingOrderTypes.IMG_LINK,
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts]: let's turn off tabindex for images here. we did that in cart (and other places)

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts [hunk 2]
```diff
@@ -1202,4 +1204,980 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
       type: TabbingOrderTypes.LINK,
     },
   ],
+  productListDesktop: [
+    {
+      type: TabbingOrderTypes.GENERIC_INPUT,
+    },
+    {
+      value: '«',
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts]: we have to go to the second page, so this button is active

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts [hunk 3]
```diff
@@ -1202,4 +1204,980 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
...
+    },
+    {
+      value: '»',
+      type: TabbingOrderTypes.LINK,
+    },
+    {
+      type: TabbingOrderTypes.CX_PRODUCT_VIEW,
+    },
+    {
+      value: 'Stores',
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts]: we should ask @developpeurweb about the order of tabbing here. following good UX, we would focus filters first, but probably the headers of filters first, then go to the right side

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts [hunk 4]
```diff
@@ -24,14 +24,21 @@ export function checkElement(tabElement: TabElement) {
...
     }
     case TabbingOrderTypes.GENERIC_NG_SELECT: {
       cy.focused().should('have.attr', 'type', 'ng-select');
       return;
     }
+    case TabbingOrderTypes.CX_PRODUCT_VIEW: {
+      cy.focused()
+        .should('have.prop', 'tagName')
+        .should('eq', 'DIV');
+      cy.focused().should('have.class', 'cx-product-layout');
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: can't we chain it to the previous `.should`s ?

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts [hunk 5]
```diff
@@ -109,6 +116,13 @@ export function checkElement(tabElement: TabElement) {
         .contains(tabElement.value);
       break;
     }
+    case TabbingOrderTypes.LI_LINK: {
+      cy.focused()
+        .should('have.prop', 'tagName')
+        .should('eq', 'LI');
+      cy.focused().contains(tabElement.value);
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: same as above

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/product-list.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
+import { checkAllElements, TabElement } from '../tabbing-order';
+import { formats } from '../../../sample-data/viewports';
+
+export function productListTabbingOrderDesktop(
+  config: TabElement[],
+  grid: boolean = false
+) {
+  cy.visit('/Brands/all/c/brands');
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/product-list.ts]: we can export this url, similarly to `testProductUrl`

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts (hunk 5 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: Removed as not in use

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/product-list.ts (hunk 6 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/product-list.ts]: Done

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts (hunk 4 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: Yes we can. Done

**Pull request merged**